src/store/index.ts

```diff
@@ -19,13 +20,26 @@ import wizardSlice, {
   selectImageTypes,
 } from './wizardSlice';
 
-export const reducer = combineReducers({
+import { isOnPremise } from '../constants';
+
+export const serviceReducer = combineReducers({
   [contentSourcesApi.reducerPath]: contentSourcesApi.reducer,
   [edgeApi.reducerPath]: edgeApi.reducer,
   [imageBuilderApi.reducerPath]: imageBuilderApi.reducer,
   [rhsmApi.reducerPath]: rhsmApi.reducer,
   [provisioningApi.reducerPath]: provisioningApi.reducer,
   [complianceApi.reducerPath]: complianceApi.reducer,
+  [blueprintsReducer.reducerPath]: blueprintsReducer.reducer,
+  notifications: notificationsReducer,
+  wizard: wizardSlice,
+  blueprints: blueprintsSlice.reducer,
+});
+
+export const onPremReducer = combineReducers({
```

Comment:
Let's remove the Insights related reducers and use only what we need for now, bare minimum to get empty blueprints table.